content/bn/service.md

```diff
@@ -0,0 +1,11 @@
+---
+title: পরিষেবা (Service)
+status: Feedback Appreciated
+category: সম্পত্তি
+---
+
+দয়া করে মনে রাখবেন যে আইটি (IT)-তে, পরিষেবার একাধিক অর্থ রয়েছে।
+এই সংজ্ঞায়, আমরা আরও ঐতিহ্যগত একটির উপর ফোকাস করব: মাইক্রোসার্ভিসের মতো পরিষেবা ৷
+কিভাবে বা এমনকি যদি পরিষেবাগুলি মাইক্রোসার্ভিসেস থেকে আলাদা হয় তা সংক্ষিপ্ত এবং বিভিন্ন লোকের ভিন্ন মতামত থাকতে পারে।
```

Comment:
Can you make a more suitable sentence instead of this one?
`কিভাবে বা এমনকি যদি পরিষেবাগুলি মাইক্রোসার্ভিসেস থেকে আলাদা হয় তা সংক্ষিপ্ত এবং বিভিন্ন লোকের ভিন্ন মতামত থাকতে পারে।`

Reply:
Okay sure